doc/api/perf_hooks.md

```diff
@@ -1063,7 +1063,7 @@ The minimum recorded event loop delay.
 added: v11.10.0
 -->
 
-* `percentile` {number} A percentile value in the range (0, 100].
+* `percentile` {number} A percentile value in the range (0, 100).
```

Comment:
Should this be `[0, 100)` based on [this validation](https://github.com/nodejs/node/blob/master/lib/internal/histogram.js#L190)?

Reply:
Right. If there's something wrong, please tell me.

Reply:
I think it should stay as is. It will throw if the value is 0 or below, or if the value is above 100. Therefore it accepts any number in the range (0, 100]. 
```suggestion
* `percentile` {number} A percentile value in the range (0, 100].
```

Reply:
I read the validation wrong earlier - this one is right (because the range is exclusive 0 and inclusive 100). Can you change `histogram.percentileBigInt` to also be `(0, 100]`. Thanks!

Reply:
@danielleadams I didn't know that expression. Thank you for telling me.